db/schema.rb

```diff
@@ -510,13 +903,447 @@
+    t.bigint "source_id", null: false
+    t.bigint "researchquestion_id", null: false
+    t.index ["researchquestion_id"], name: "projects_source_research_question_research_question_id_b8a06e2f"
+    t.index ["source_id", "researchquestion_id"], name: "projects_source_research_source_id_research_quest_928025bc_uniq", unique: true
+    t.index ["source_id"], name: "projects_source_research_question_source_id_451782de"
+  end
+
+  create_table "projects_source_topics_covered", force: :cascade do |t|
+    t.bigint "source_id", null: false
+    t.bigint "topics_id", null: false
+    t.index ["source_id", "topics_id"], name: "projects_source_topics_c_source_id_topics_id_b59edf45_uniq", unique: true
+    t.index ["source_id"], name: "projects_source_topics_covered_source_id_d120fe6e"
+    t.index ["topics_id"], name: "projects_source_topics_covered_topics_id_83f74664"
+  end
+
+  create_table "projects_topics", force: :cascade do |t|
+    t.string "name", limit: 200, null: false
+  end
+
+  create_table "resources_category", force: :cascade do |t|
+    t.string "name", limit: 200, null: false
+    t.bigint "parent_category_id"
+    t.text "svg_code", null: false
+    t.timestamptz "created_at", null: false
+    t.timestamptz "updated_at", null: false
+    t.index ["parent_category_id"], name: "resources_category_parent_category_id_6fffa3b5"
+  end
+
+  create_table "resources_resource", force: :cascade do |t|
+    t.text "name", null: false
+    t.string "address", limit: 200
+    t.string "city", limit: 200
+    t.string "state", limit: 200
+    t.string "zip", limit: 200
+    t.jsonb "details_html"
+    t.bigint "parcel_id"
+    t.timestamptz "created_at", null: false
+    t.timestamptz "updated_at", null: false
+    t.bigint "list_html_id"
+    t.bigint "refresh_id", null: false
+    t.index ["list_html_id"], name: "resources_resource_list_html_id_93af9cdb"
+    t.index ["parcel_id"], name: "resources_resource_parcel_id_d8e5f681"
+    t.index ["refresh_id"], name: "resources_resource_refresh_id_a263997c"
+  end
+
+  create_table "resources_resource_categories", force: :cascade do |t|
+    t.bigint "resource_id", null: false
+    t.bigint "category_id", null: false
+    t.index ["category_id"], name: "resources_resource_categories_category_id_3944a462"
+    t.index ["resource_id", "category_id"], name: "resources_resource_categ_resource_id_category_id_934a21a6_uniq", unique: true
+    t.index ["resource_id"], name: "resources_resource_categories_resource_id_f2f988e9"
+  end
+
+  create_table "resources_resourcelisthtml", force: :cascade do |t|
+    t.text "html", null: false
+    t.timestamptz "created_at", null: false
+    t.boolean "parsed", null: false
+    t.jsonb "request", null: false
+    t.timestamptz "updated_at", null: false
+    t.bigint "refresh_id", null: false
+    t.index ["refresh_id"], name: "resources_resourcelisthtml_refresh_id_f68a7098"
+  end
+
```

Comment:
This is a big schema change that is not related to this PR. Let's drop this so that our schema file stays clean 🧹